src/modules/scores.js

```diff
@@ -9,4 +17,83 @@ const refreshScores = () => {
   window.location.reload();
 };
 
-refreshButton.addEventListener('click', refreshScores);
\ No newline at end of file
+// Add new score to the list and save them into localStorage
+const addScore = () => {
+  const nameInput = document.querySelector('#name').value;
+  const scoreInput = document.querySelector('#score').value;
+
+  if (nameInput && scoreInput && scoreInput !== 0 && scoreInput > 0) {
+    // Save the data into the API
+    const myHeaders = new Headers();
+    myHeaders.append('Content-Type', 'application/json');
+
+    const raw = JSON.stringify({
+      user: nameInput,
+      score: parseInt(scoreInput, 10),
+    });
+
+    const requestOptions = {
+      method: 'POST',
+      headers: myHeaders,
+      body: raw,
+      redirect: 'follow',
+    };
+
+    fetch(`${urlAPI}${idAPI}/scores/`, requestOptions)
+      .then((response) => response.text())
+      .catch((error) => error);
+
+    // Reset the form and reload the page
+    setTimeout(() => {
+      document.querySelector('.add-form').reset();
+      window.location.reload();
+    }, 2000);
+  } else if (scoreInput === 0 || scoreInput < 0) {
+    errorElement.innerHTML = '<p>Score must be greater than 0</p>';
```

Comment:
- [Optional] It'll be nice to handle your validations properly once more. A wrong error message was displayed when the scoreInput is zero as you can see on the screenshot below:
![Screenshot from 2023-06-23 22-27-46](https://github.com/Ruthmy/leaderboardProject/assets/63087491/899dbfdb-caff-4c86-9fcc-479b5b637890)

Reply:
Hi @FrederickMih
Thank you very much for your feedback!
I already made the changes to better evaluate the errors, scenarios like this already give a better description of the error. Thanks.